modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -110,6 +110,11 @@
 			"jspf"
 		]
 	},
+	{
+		"from": "regex:(?<autoBatch>AutoBatchPreparedStatementUtil\\.autoBatch\\(\\s*)(?<connection>connection).prepareStatement\\((?<message>\".*\")\\)\\)",
```

Comment:
`(?<connection>connection)` should be named `(?<variableName>\w+)` - it's a variable.
But we have a problem here. If was only this would not be a L size. In the ticket we have also:

'Needs to deal with cases where the statement it’s mount outside the method call and filled as a parameter as well.'

Every case like this won't be catch by this pattern above.